### PR TITLE
Course page

### PR DESCRIPTION
**Pull request by kuouu**
Page will crush when it cannot find relative image.
Also need to wait for complete course data.

### CODE REVIEW

File: src/components/course/index.jsx [hunk 1]
```diff
@@ -1,12 +1,151 @@
...
+              <b>演講者：</b>
+              {course.presenter}
+            </Row>
+            <Row>{course.description}</Row>
+          </Information>
+          <Source
+            src={
+              images.find((e) =>
+                e.default.includes(course.pictureName.split('.')[0])
+                  .default
```
Comment (coliver0221) [inline src/components/course/index.jsx]: Missing a `)` here.

**Review by coliver0221 — CHANGES_REQUESTED** [review]

**Comment by coliver0221** [comment]
Please fix these images' name:
`4-10.png` -> `4-I10.png`
`4-11.png` -> `4-I11.png`

File: src/components/course/index.jsx (hunk 1 above)
Comment (kuouu) [reply src/components/course/index.jsx]: Done in a9fb157065ef82c6b9c60c6702a620bb0bf1389b

**Review by coliver0221 — APPROVED** [review]

**Comment by coliver0221** [comment]
LGTM.

**Pull request merged**